paddlenlp/trainer/training_args.py

```diff
@@ -603,6 +605,10 @@ class TrainingArguments:
         default=False,
         metadata={"help": "Whether to use paddle.async_save instead of paddle.save."},
     )
+    enable_timer: Optional[bool] = field(
```

Comment:
建议
```suggestion
    skip_profiler_timer: Optional[bool] = field(
```
然后默认值为True，用户传入 False 时候，打开timer。

Reply:
之前有参数`--skip_memory_metrics` 也是这么设置的

Reply:
fixed